### ISSUE TITLE
False positive for MD011 when footnote is next to a link.

### ISSUE THREAD

**Issue by brainsik** [rvben/rumdl#147]
Example document:

```md
# Header

Some text with [a link](https://example.com/)[^ft].

[^ft]: Note.
```

Result:

```txt
$ rumdl check --isolated example.md
example.md:3:24: [MD011] Reversed link syntax: use [^ft](https://example.com/) instead [*]

Issues: Found 1 issues in 1 file (0ms)
Run `rumdl fmt` to automatically fix 1 of the 1 issues
```

Using rumdl 0.0.174.

**Comment by rvben** [comment]
Hi @brainsik,

Thanks for raising this issue. Hadn't seen this one before, but clearly a false positive.
The fix has been released as part of 0.0.175.

Thank you for helping me remove bugs from rumdl! 🚀
